extensions/ql-vscode/package.json

```diff
@@ -1084,6 +1096,10 @@
         {
           "command": "codeQL.previewQueryHelp",
           "when": "resourceExtname == .qhelp && isWorkspaceTrusted"
+        },
+        {
+          "command": "codeQL.gotoQL",
+          "when": "editorLangId == ql-summary"
```

Comment:
Do you need to add `"enablement": "codeql.hasQLSource"` here so that the command is only enabled when the context variable is true? Not sure if it works the way you have it. Might be fine...just not sure.

Reply:
Shall we also gate this behind the canary flag?